Source/WebCore/svg/SVGUseElement.cpp

```diff
@@ -84,15 +84,22 @@ void SVGUseElement::attributeChanged(const QualifiedName& name, const AtomString
 {
     SVGParsingError parseError = NoError;
 
-    if (name == SVGNames::xAttr)
+    switch (name.nodeName()) {
```

Comment:
Don’t we need to include `NodeName.h` here?

Reply:
Already included.